recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Default set of sounds for AsteroidOS under CC0"
+HOMEPAGE = "https://github.com/AsteroidOS/asteroid-sounds"
+LICENSE = "CC0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
```

Comment:
The license checksum isn't correct for what's provided in the repo. This should be `65d3616852dbf7b1a6d4b53b00626032`:
```
LIC_FILES_CHKSUM = "file://LICENSE;md5=65d3616852dbf7b1a6d4b53b00626032"
```

Reply:
ugh, yeah, i did generate the md5sum for the meta-asteroid LICENSE files. Bravo to myself.